### ISSUE TITLE
Pytorch optimizer 与 learning rate - 会飞的咸鱼

### ISSUE THREAD

**Issue by TankZhouFirst** [TankZhouFirst/TankZhouFirst.github.io#83]
https://tankzhou.cn/2019/08/20/optimizer-%E4%B8%8E-learning-rate/

**Comment by TankZhouFirst** [comment]
沙发，存档
